### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by JIAJZHAN-Kairos**
The documents of how to use IntOGen on Seqera and how to deploy this pipeline on Seqera.

### CODE REVIEW

File: Usage.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+Cohort file names, as well as the fields mentioned above must not contain dots.
+The way to provide those values is through [OpenVariant](https://github.com/bbglab/openvariant) , a comprehensive Python package that provides different functionalities to read, parse and operate different multiple input file formats (e. g. tsv, csv, vcf, maf, bed). Whether you are planning to run single or multiple cohorts, you would need to provide an annotation file in yaml format to specify the above mentioned structure required by IntOGen. 
+
+What you need to do is copy the `s3://org.umccr.nf-tower.general/intogen-plus-2024/test/` metadata.yaml* to your input folder (the metadata.yaml and your input data need to be in the same folder) and modify these parameters as necessary.
+
+Example:
+```
+pattern:
+
+- 'combined-somatic-PASS.tsv' # The file name of your inpute data
```
Comment (skanwal) [inline Usage.md]: typo in "inpute"

File: Usage.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+By default this pipeline outputs 4 files:
+
+* *`cohorts.tsv`:* summary of the cohorts that have been analyzed
+* *`drivers.tsv`:* summary of the results of the driver discovery by cohort
+* *`mutations.tsv`:* summary of all the mutations analyzed by cohort
+* `unique_drivers.tsv`: information on the genes reported as drivers (in any cohort)
+* `unfiltered_drivers.tsv`: information on the filters applied to the post-processing step: from the output of the combination to the final set of driver genes.
+Those files can be found in the path indicated with the --output options.
+The output of each tools can be found in the step folder.
+#### Output Explanation
```
Comment (skanwal) [inline Usage.md:84]: this can be done as a table too (just a suggestion)

File: Usage.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
+# How to Use the IntOGen Pipeline on Seqera
```
Comment (skanwal) [inline Usage.md:1]: this is good. I suggest adding a run example with TCGA data as input and walking through each step around how you run it and which links to use.

**Review by skanwal — APPROVED** [review]
Thanks, this LGTM. I have few minor suggestions.

File: Usage.md (hunk 3 above)
Comment (JIAJZHAN-Kairos) [reply Usage.md:1]: Thank you for the suggestions!

**Pull request merged**